File: LP_Python/loop_looping/listas/loop_2.py
X = int(0)
I = int(0)
MD = float(0)

while True:
    N = int(input("Digite numeros positivos: "))
    if N<=0:
        break
    X = N + X
    MD+=1
    I+=1

MD=X/I
print("media= %.2f\nsoma= %d"%(MD,X))

File: LP_C++/lista_loop/loop_2.cpp

#include <stdio.h>
#include <stdlib.h>

int main(int argc, char const *argv[])
{
    int soma = 0, n, count = 0;
    float md;

    do {
        printf("Digite números positivos: \n");
        scanf("%d", &n);
        if (n > 0) {
            soma += n;
            count++;
        }
    } while (n > 0);

    md = (float) soma / count;
    printf("média = %.2f\nsoma = %d\n", md, soma);

    return 0;
}